src/ocm/components/clusterDetail/ClusterProperties.tsx

```diff
@@ -37,23 +37,26 @@ type ClusterPropertiesProps = {
   externalMode?: boolean;
 };
 
-const getNetworkType = (clusterNetworkType: Cluster['networkType']): string => {
+export const getNetworkType = (clusterNetworkType: Cluster['networkType']): string => {
   let networkType: string;
   clusterNetworkType === NETWORK_TYPE_SDN
     ? (networkType = 'Software-Defined Networking (SDN)')
     : (networkType = 'Open Virtual Network (OVN)');
   return networkType;
 };
 
-const getManagementType = (isUserManagementType: boolean | undefined): string => {
+export const getManagementType = ({ userManagedNetworking }: Cluster): string => {
   let managementType: string;
-  isUserManagementType
+  userManagedNetworking
     ? (managementType = 'User-Managed Networking')
     : (managementType = 'Cluster-managed networking');
   return managementType;
 };
 
-const getDiskEncryptionEnabledOnStatus = (diskEncryption: DiskEncryption['enableOn']) => {
+export const getStackType = (cluster: Cluster): string =>
+  isDualStack(cluster) ? 'Dual-stack' : 'IPv4';
+
+export const getDiskEncryptionEnabledOnStatus = (diskEncryption: DiskEncryption['enableOn']) => {
```

Comment:
This looks like it could be a React Component instead of a function. Feel free to ignore :)

Reply:
Keeping this one as is.